guides/source/developers/extensions/writing-extensions.html.md

```diff
@@ -0,0 +1,67 @@
+# Writing a New Solidus Extension
+
+Sometimes building a site powered by Solidus will require customization.
+Solidus extensions are Rails engines that allow you to extend or change the
+functionality of the Solidus platform.
+
+## Find existing extensions
+
+Before you write a new Solidus extension you should look for an existing
+extension that meets your needs. Solidus maintains an official list of
+[supported extension](http://extensions.solidus.io), but you can also use
```

Comment:
Small typo. I think this should say `[supported extensions]` rather than `[supported extension]`

Reply:
Oops! Good catch.